Day-2/day-2-2.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const text = fs.readFileSync("./input.txt", "utf-8");
+const textByLine = text.split("\n");
+const arrayLength = textByLine.length;
+
+function getResult(lines, array){
+    let distance = 0
+    let depth = 0
+    let aim = 0
+    for(let i=0; i < array; i++){
+        let words = lines[i].split(" ")
```

Comment:
Awesome that you're starting to use `const` and `let`. This is actually tricky since you're in a `for` loop, you create a new scope on each pass of the loop. That means any variables you declare get wiped from the previous loop and recreated on the next loop. So `words` doesn't actually ever get reassigned inside the loop, so it could be `const`. This goes for any other variables inside the loop that don't get reassigned.